templates/blocks/blok_dossier.twig

```diff
@@ -1,38 +1,49 @@
-<div class="bg-gray-800 text-white">
+<div class="bg-gray-800 text-white my-4">
```

Comment:
`my` is hier niet echt nodig, en is raar als je twee blokken met achtergrond boven elkaar zet